kuma/core/tests/test_form_fields.py

```diff
@@ -27,11 +27,11 @@ def test_xx_YY_locale(self):
         num = _format_decimal(1234.567)
         assert '1,234.567' == num
 
-    def test_fy_NL_locale(self):
+    def test_pt_BR_locale(self):
```

Comment:
NOTE: `babel` doesn't know about the `pt-XX` locales. If it did, the decimal would be formatted as `1.234,567` rather than `1,234.567`.